pkg/cmd/server/api/helpers.go

```diff
@@ -12,6 +13,82 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/crypto"
 )
 
+func GetMasterFileReferences(config *MasterConfig) []*string {
```

Comment:
Having a method called ResolveMasterConfigPaths that takes a masterconfig pointer will make it more obvious that mutation is happening.  It's not clear from calling code that these calls work together to mutate the underlying contents.